static/pages/game/game.js

```diff
@@ -462,6 +463,7 @@ class Game {
                 this.attackUnit(move.attackerTileID, move.targetTileID)
             }
         }
+        ui.UpdateMinimapUnits()
```

Comment:
```suggestion
        setTimeout(() => ui.UpdateMinimapUnits(), 2000)
```
Timeout to prevent updating map before units finish moving